deploy/charts/cert-manager/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.servicemonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "cert-manager.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "cert-manager.name" . }}
+    chart: {{ template "cert-manager.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    prometheus: {{ .Values.prometheus.servicemonitor.prometheusInstance }}
+spec:
+  jobLabel: {{ template "cert-manager.fullname" . }}
+  selector:
+    matchLabels:
+      app: {{ template "cert-manager.name" . }}
+      release: {{ .Release.Name }}
+  endpoints:
```

Comment:
I think you need a [namespaceSelector](https://github.com/helm/charts/blob/master/stable/prometheus-operator/templates/prometheus-operator/crd-servicemonitor.yaml#L224) definition. Could be wrong about that

Reply:
Good catch @ev1lm0nk3y, recuded to 30s!

Reply:
Is a namespaceSelector required as well? I'm not familiar enough with what's required with the prometheus operator 😄

Reply:
I believe it is. let me try and create one without a namespaceSelector and see what happens.

Reply:
namespaceSelector is not required and default is the namespace that the ServiceMonitor is deployed to which is the same as the rest of the Helm Chart and the Cert-Manager which is correct behaviour 😄